### ISSUE TITLE
Application: Document Collaboration on IPFS 

### ISSUE THREAD

**Issue by flyingzumwalt** [ipfs/ipfs#238]
ie. hackmd and cryptpad on IPFS

Work in progress - please contribute. See #230.

**Comment by chrismatthieu** [comment]
Checkout http://philes.co

**Comment by tapaswenipathak** [comment]
Hello folks, Is there any work done on this ticket? Any PR or research?

**Comment by Stebalien** [comment]
Yes, actually: https://peerpad.net/. We should close this as implemented.

And thanks for helping cleaning up old issues!

**Comment by Clay-Ferguson** [comment]
My project is SubNode.org. It's a content editing platform that will *yes soon I promise* be able to publish JSON or HTML directly to IPFS, from the web app.  In my opinion the "New Social [media] Web" (web 3.0) should be built on this type of architecture that I've created.

**Comment by jessicaschilling** [comment]
Note: Discussion on applications of IPFS are happening over in the [IPFS Forums](https://discuss.ipfs.io/c/ecosystem/applications-of-ipfs/) now ... please continue the discussion there! 

This issue is being moved over to the archived repo https://github.com/ipfs/apps/ for reference.
